site/src/pages/index.astro

```diff
@@ -496,6 +508,10 @@ import MetaLayout from "@/layouts/MetaLayout.astro";
           which is insufficient to disambiguate for colorblind users, as some
           controls feature the same icons distinguished only by color.
         </dd>
+        <dt>1.4.1 Use of Color</dt>
+        <dd>The controls are color coded and instructions depend on them. </dd>
+        <dt>1.4.10: Reflow</dt>
+        <dd>The controls disappear when magnified.</dd>
```

Comment:
This is technically due to the mobile breakpoint being hit, which explicitly hides the controls (it's not that it's being scrolled off-screen). Is reflow still accurate in this case?